ansible/roles/master/tasks/main.yml

```diff
@@ -12,29 +14,21 @@
   tags:
     - binary-update
 
-- name: write the config file for the api server
```

Comment:
Will master non-ha work with the lines you removed in this file? If not, then this pr will get blocked. Keep in mind support for centos 7 as well. this will need to be tested in vagrant.